tools/buildsys/src/main.rs

```diff
@@ -124,6 +126,7 @@ fn build_package(args: BuildPackageArgs) -> Result<()> {
     let manifest_file = "Cargo.toml";
     let manifest_path = args.common.cargo_manifest_dir.join(manifest_file);
     println!("cargo:rerun-if-changed={}", manifest_file);
+    println!("cargo:rerun-if-changed={}", EXTERNAL_KIT_METADATA);
```

Comment:
`cargo` will always rerun a build if told to `rerun-if-changed` on a file that doesn't exist. So you must either emit this conditionally (if the file exists), or ensure that `twoliter` always creates at least an empty file.

Reply:
This occurs when you run twoliter fetch, there should always be that file with at minimum the sdk info. We could also do this somewhere else too if you have a good idea where?